doc/configure.md

```diff
@@ -192,7 +201,7 @@ IP Tunneling
 
 IP Tunneling will allow you to connect from the cjdns network to another outside network. This is still a work in-progress; although it does function, it requires a bit of manual configuration on both ends to make it useful.
 ````javascript
-        // System for tunneling IPv4 and ICANN IPv6 through cjdns.
+        // System for tunneling IPv4 and ICANN IPv6 through cjdn which will eventually be merged to master..
```

Comment:
Yeah no idea. There's a lot of crap in the docs repo -- we need to be more serious about it. Stuff like the Emercoin repo is just irelevant. Also we should just ruthlessly delete anything that's only slightly outdated.